config/checkstyle.xml

```diff
@@ -8,7 +8,7 @@
 	<!-- Checks whether files end with a new line. -->
 	<!-- See http://checkstyle.sourceforge.net/config_misc.html#NewlineAtEndOfFile -->
 	<module name="NewlineAtEndOfFile">
-		<property name="severity" value="info"/>
+		<property name="severity" value="warning"/>
```

Comment:
Or "ignore", thoughts?

Reply:
Agree. I'm going to add `ignore` to `NewlineAtEndOfFile` and `error` for trailing spaces.